srcs/sphere/sphere_color.c

```diff
@@ -2,4 +2,9 @@
 #include "vector.h"
 #include "ray.h"
 #include "object.h"
+#include "color.h"
 
+int	convert_rgb(t_rgb color)
+{
+	return ((color.r << 16) | (color.g << 8) | color.b);
+}
```

Comment:
define SHIFT なるものがありました…！
```suggestion
	return ((color.r << RED_SHIFT) | (color.g << GREEN_SHIFT) | color.b);
```

Reply:
ﾎﾝﾏﾔ---!
ありがとうございます！

Reply:
なんかズレてたっぽいので #22 で修正しましたっ

Reply:
面目ないです…:tired_face: